package.json

```diff
@@ -71,19 +72,19 @@
     "devDependencies": {
         "@nestjs/cli": "^9.1.4",
         "@nestjs/schematics": "^9.0.3",
-        "@nestjs/testing": "^9.1.2",
+        "@nestjs/testing": "^9.1.4",
         "@types/bull": "^3.15.9",
         "@types/cache-manager": "^4.0.2",
         "@types/express": "^4.17.14",
-        "@types/jest": "29.1.1",
+        "@types/jest": "29.1.2",
         "@types/joi": "^17.2.3",
         "@types/moment": "^2.13.0",
-        "@types/node": "^18.8.1",
+        "@types/node": "^18.8.3",
         "@types/socket.io": "^3.0.2",
         "@types/supertest": "^2.0.12",
         "@typescript-eslint/eslint-plugin": "^5.39.0",
         "@typescript-eslint/parser": "^5.39.0",
-        "eslint": "^8.24.0",
+        "eslint": "8.22.0",
```

Comment:
Why downgrading eslint?

Reply:
Webstorm issue that don't recognize latest version of ESLint